src/controllers/wallet/wallet.controller.js

```diff
@@ -341,6 +341,7 @@ async function decodeTx(req, res) {
             token: utxo.token,
             value: utxo.value,
             tokenData: utxo.token_data,
+            token_data: utxo.token_data,
```

Comment:
Add explanation.

Reply:
Done at: fd88de218d00d2f5f86d1873fc3b5d18840e4d76

Reply:
We definitely should define a consistent pattern throughout our code. I will open an issue for us to discuss it.

Reply:
Here is the issue: https://github.com/HathorNetwork/hathor-wallet-lib/issues/562